packages/vscode-extension/src/debug/teamsfxTaskHandler.ts

```diff
@@ -110,9 +112,25 @@ async function onDidEndTaskProcessHandler(event: vscode.TaskProcessEndEvent): Pr
       try {
         if (cwd !== undefined && event.exitCode !== undefined && event.exitCode !== 0) {
           npmInstallLogInfo = await getNpmInstallLogInfo();
+          showError(new SystemError(
+            npmInstall,
+            npmInstallErrorMessage,
+            task.name,
+            JSON.stringify(npmInstallLogInfo),
+            issueLink,
+            npmInstallLogInfo
+          ));
         }
-      } catch {
+      } catch (e) {
         // ignore any error
+        showError(new SystemError(
```

Comment:
This is internal error when getting log file, no need to show the same as above.
Telemetry could be a good improvement, but can be in later PR.

Reply:
Already remove inner error show error logic.